pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java

```diff
@@ -214,10 +214,10 @@ public class ElasticSearchConfig implements Serializable {
 
     @FieldDoc(
             required = false,
-            defaultValue = "5",
-            help = "Idle connection timeout to prevent a read timeout."
+            defaultValue = "30000",
+            help = "Idle connection timeout to prevent a connection timeouts."
     )
-    private int connectionIdleTimeoutInMs = 5;
+    private int connectionIdleTimeoutInMs = 30000;
```

Comment:
I wonder if this config should be validated against [bulkFlushIntervalInMs](https://github.com/apache/pulsar/blob/09504017526308fd83f480aa8cb502e1cfe0f633/pulsar-io/elastic-search/src/main/java/org/apache/pulsar/io/elasticsearch/ElasticSearchConfig.java#L189) when bulk API is enabled - something like `connectionIdleTimeoutInMs > 2 * bulkFlushIntervalInMs` because it seems the connection will set idle by design in-between flushes

Reply:
I think it'd be fine to let the connection get closed in that scenario. My main goal here is to make sure we have working defaults.

Reply:
Yeah I understand both comments are probably outside scope of this PR: First step is to make have working defaults, and later maybe make them foolproof...